Source/UI/SecurityCodeComponent/SecurityCodeComponent.swift

```diff
@@ -35,23 +45,25 @@ extension SecurityCodeComponent {
                         isSecurityCodeValid: @escaping (Bool) -> Void) {
     self.configuration = configuration
     self.isSecurityCodeValid = isSecurityCodeValid
-
-    self.cardValidator = CardValidator(environment: configuration.environment.checkoutEnvironment)
-
+    
+    cardValidator = CardValidator(environment: configuration.environment.checkoutEnvironment)
+    checkoutAPIService = CheckoutAPIService(publicKey: configuration.apiKey,
```

Comment:
If I understand this right, The SecurityCodeComponent.swift requires a checkoutAPIService and cardValidator. And if merchants were to create a card token then there would also be another CheckoutAPIService instance created by the merchant app using the API key and environment. Also here we are passing the CardValidator which requires environment. Not a blocker for this PR but wanted to understand, Is there a way to improve this? 
Also

Reply:
Definitely a good catch and question. We can have a look if there is a way to use the same instance. 

But also, I'm not so sure of the balance of added-value/effort would be high of that improvement. `CheckoutAPIService` is cheap. Nothing big or memory consuming or complicated. Creating it many times shouldn't be a problem and trying to have just one instance of it might be over optimising the codebase. 

In any case, noted and will have a look. Thanks. 
@deepesh-vasthimal-cko